x-pack/plugins/search_inference_endpoints/server/lib/delete_inference_endpoint.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ElasticsearchClient } from '@kbn/core/server';
+
+export const deleteInferenceEndpoint = async (
+  client: ElasticsearchClient,
+  type: string,
+  id: string
+): Promise<void> => {
+  await client.transport.request({
```

Comment:
we should return the value even if we're not using it

Reply:
Updated